homeassistant/components/yolink/binary_sensor.py

```diff
@@ -104,6 +104,9 @@ def __init__(
         """Init YoLink Sensor."""
         super().__init__(coordinator, device)
         self.entity_description = description
+        self._attr_is_on = self.entity_description.value(
```

Comment:
I suggest we call `self._handle_coordinator_update` in `async_added_to_hass` in the base entity class instead.

Reply:
that's right